docs/time.md

```diff
@@ -375,7 +375,7 @@ This method overrides `Object.prototype.toLocaleString()` to provide a human-rea
 
 The `locales` and `options` arguments are the same as in the constructor to [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat).
 
-> **NOTE**: Unlike in [`Temporal.Absolute.prototype.toLocaleString()`](./absolute.html#toLocaleString), `locales.timeZone` will have no effect, because `Temporal.Time` carries no time zone information and is just a wall-clock time.
+> **NOTE**: Unlike in [`Temporal.Instant.prototype.toLocaleString()`](./absolute.html#toLocaleString), `locales.timeZone` will have no effect, because `Temporal.Time` carries no time zone information and is just a wall-clock time.
```

Comment:
This reminds me, I'd suggest renaming `absolute.md` to `instant.md`.
```suggestion
> **NOTE**: Unlike in [`Temporal.Instant.prototype.toLocaleString()`](./instant.html#toLocaleString), `locales.timeZone` will have no effect, because `Temporal.Time` carries no time zone information and is just a wall-clock time.
```